docker/environment.yml

```diff
@@ -3,8 +3,9 @@ channels:
   - conda-forge
 dependencies:
   - python=3.9.*
```

Comment:
If we're upgrading the image anyway, is it worth moving to a more recent version of python as well? Most of the python scientific ecosystem uses the policy of supporting only the 3 most recent python versions. 3.9 is currently the third most recent version. Just a thought.

Reply:
Good point Geoff, thanks - worth seeing if we can upgrade; I can look into this